speakers.md

```diff
@@ -1,52 +1,66 @@
-# Submitting Talk
+# Submitting A Talk
 
-## Who can submit talks to BrooklynJS
-We welcome & encourage _everyone_ to submit talk!
-Whether you have been writing JavaScript for 20 years or few days, experienced or 1st time* speaker, we want to hear that you are excited about.
+## Who can submit talks to BrooklynJS?
+We welcome & encourage _everyone_ to submit a talk!
+Whether you have been writing JavaScript for 20 years or few days, experienced or 1st time speaker*, we want to hear what you are excited about.
 
-*Our speaker alumni includes Jenna Zeigen (@zeigenvector), Drew Petersen (@KirbySaysHi), Sarah Saltrick Meyer (@meyerini), Byron Hulcher (@hypirlink) & many more who gave their 1st talk at BrooklynJS then started speaking at different meetups and conferences. If you have never spoken before we would love you to submit talk!
+_*Our speaker alumni includes Jenna Zeigen (@zeigenvector), Drew Petersen (@KirbySaysHi), Sarah Saltrick Meyer (@meyerini), Byron Hulcher (@hypirlink) & many more who gave their first talk at BrooklynJS, then started speaking at different meetups and conferences. BrooklynJS is very first time speaker friendly!_
```

Comment:
Should we remove Jenna now that she's an organizer?

Reply:
Sounds good to me! 👍